src/main/java/com/chariotsolutions/meetupnotifier/AppBeanImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.chariotsolutions.meetupnotifier;
+
+
+import com.google.api.services.calendar.model.Event;
+
+import com.chariotsolutions.meetupnotifier.meetup.api.GoogleConverter;
+import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQuery;
+import com.chariotsolutions.meetupnotifier.meetup.api.MeetupQueryExecutor;
+
+import com.chariotsolutions.meetupnotifier.meetup.api.messages.Result;
+import com.chariotsolutions.meetupnotifier.meetup.api.messages.Results;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@EnableScheduling
+public class AppBeanImpl {
+
+  private static final Logger logger = LoggerFactory.getLogger(AppBeanImpl.class);
+
+  private String[] groups;
+
+  @Scheduled(fixedRate = 60000L, initialDelay = 4000L)
```

Comment:
Can you externalize the configuration of this bean / scheduling in `context.xml` ?